tests/foreman/cli/test_remoteexecution.py

```diff
@@ -650,6 +654,15 @@ def test_positive_run_reccuring_job(self, rex_contenthost):
         rec_logic = RecurringLogic.info({'id': result['recurring-logic-id']})
         assert rec_logic['state'] == 'finished'
         assert rec_logic['iteration'] == '2'
+        # 2129432
+        rec_logic_keys = rec_logic.keys()
+        assert 'action' in rec_logic_keys
+        assert 'last-occurence' in rec_logic_keys
```

Comment:
Looks like prt job is failing due to spelling. should be 'last-occurrence'

Reply:
facepalm